tests/test_create_config.py

```diff
@@ -1,18 +1,22 @@
 from pathlib import Path
+import os
 import subprocess
 
 from .utils import python
 
 
-def test_create_config():
-    result = subprocess.run([python(), "-m", "universum", "init"], capture_output=True, check=True)
+def test_create_config(tmpdir):
+    my_env = os.environ.copy()
```

Comment:
In my opinion, the following construct is cleaner:
```python
test_environment=dict(os.environ, PYTHONPATH=os.getcwd())
```
Also note the better variable name.

Reply:
agree